config/jobs/kubernetes/sig-cloud-provider/azure/generate.sh

```diff
@@ -37,7 +37,7 @@ generate_presubmit_annotations() {
 EOF
 }
 
-readonly kubekins_e2e_image="gcr.io/k8s-testimages/kubekins-e2e:v20210428-a1a20d1"
+readonly kubekins_e2e_image="gcr.io/k8s-testimages/kubekins-e2e:v20171002-6e8d729f"
```

Comment:
Is there any reason why this is changed to a really old image? We wanted to autobump the kubekins-e2e test image version in a bash script that helps us generate jobs

Reply:
Just took a look. I believe the autobumper must bump to the image with the "latest" tag applied to it. The kubekins-e2e image with the "latest" tag is v20171002-6e8d729f.

Reply:
That makes sense. I will modify my script accordingly